language/vm/vm-runtime/vm-runtime-types/src/value.rs

```diff
@@ -491,6 +492,41 @@ impl From<Value> for VMResult<GlobalRef> {
     }
 }
 
+pub(crate) trait ValueRef<T> {
```

Comment:
It seems to me that `NativeStructRef` is a better name here? `ValueRef` sounds a bit confusing to me,

Reply:
I would say `ValueRef` is the correct name. The point of this trait is to generalize the concept of a Move reference, and the referee is not limited to a `NativeStruct` -- It can be `U64`, `Bool` or any other variant in the `ValueImpl`. Right now I only implemented the one for `NativeVector`, but more can be added on an as-needed basis.

Reply:
I see! Sounds good to me then!